src/gui/advancedsettings.cpp

```diff
@@ -431,6 +468,9 @@ void AdvancedSettings::loadAdvancedSettings()
     // Super seeding
     checkBoxSuperSeeding.setChecked(session->isSuperSeedingEnabled());
     addRow(SUPER_SEEDING, tr("Strict super seeding"), &checkBoxSuperSeeding);
+    item(SUPER_SEEDING, PROPERTY)->setToolTip(QString("<body><p>%1</p></body>")
+            .arg(tr("When this is set to true, a piece has to have been forwarded "
+                    "to a third peer before another one is handed out")));
```

Comment:
@thalieht:  But what happens if this is set to `false`?

Reply:
The opposite is implied?

Reply:
No restrictions whatsoever, give as many pieces as possible without caring whether the recipients will give back anything to the swarm.

Reply:
@thalieht:  Hmm.  How about, for your second line, you change it to read: 

>to a **certain number of peers (three)** before another **piece** is handed out 

That should clarify the nature of this setting.

Reply:
It means that the seeder must see the piece he just gave to someone, has been forwarded to someone else in the swarm (a third peer) _after_ (not sure about this one but it makes sense) the seeder gave the piece. It's not three other peers. How can this be made more clear (really asking)?

Reply:
@thalieht:  

>If this is set to true, peers must forward received pieces before being sent new ones (in super-seeding mode)

Reply:
ok changed.